sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveStrategies.scala

```diff
@@ -145,15 +146,27 @@ class DetermineTableStats(session: SparkSession) extends Rule[LogicalPlan] {
  * `PreprocessTableInsertion`.
  */
 object HiveAnalysis extends Rule[LogicalPlan] {
+  private def checkFieldNames(table: CatalogTable): Unit = {
```

Comment:
This one can be consolidated with `DataSourceAnalysis.checkFieldNames`

Move the consolidated one to `DDLUtils`

Reply:
The logic is slightly different.

**HiveStrategies**
```
  private def checkFieldNames(table: CatalogTable): Unit = {
    val serde = table.storage.serde
    if (serde == HiveSerDe.sourceToSerDe("orc").get.serde) {
      OrcFileFormat.checkFieldNames(table.dataSchema)
    } else if (serde == HiveSerDe.sourceToSerDe("parquet").get.serde) {
      ParquetSchemaConverter.checkFieldNames(table.dataSchema)
    }
  }
```

**DataSourceStrategies**
```
  private def checkFieldNames(table: CatalogTable): Unit = {
    table.provider.get.toLowerCase(Locale.ROOT) match {
      case "parquet" =>
        ParquetSchemaConverter.checkFieldNames(table.dataSchema)
      case "orc" =>
        OrcFileFormat.checkFieldNames(table.dataSchema)
      case _ =>
    }
  }
```

Reply:
Hmm. Yep. I'll try~

Reply:
It's consolidated into `DDLUtils`. Thanks!